src/bikeracks/static/main.js

```diff
@@ -60,6 +72,15 @@ $(document).ready(function() {
     $('#removalModal').on('hidden.bs.modal', function () {
         $(".removal-btn:first-child").text("Duplicate");
     })
+
+    //Removal suggestion Modal click handler
+    $('#removalModal').on('show.bs.modal', function (event) {
+        const button = $(event.relatedTarget),
+            rack_id = button.data('rack_id');
+        // add the rack_id as an attribute on send button in the modal so that
+        // if user clicks on send, the rack_id is sent in the request
+        $('#suggestRemoval').attr('data-rack_id', rack_id);
```

Comment:
Isn't the rack id already available elsewhere? You construct the trash-can element by baking in the rack-id, which is available here via `button.data('rack_id')`, so why does it need to be attached to `#suggestRemoval`?

Reply:
I commented out the entire function because I will be working on this in the next pull request.

Reply:
In general you want pull requests to be valid snapshots of the project. It would take a bit of advanced git magic to split this up, which you probably don't want to figure out right now, so go ahead with this. Just something to keep in mind in the future.